### PR TITLE
Replace PS1 cutout image with DESI DR8

### LINKED ISSUE #217: Use DESI DR8 for postage stamps
Instead of PS1, the default postage stamp to grab should be DESI DR8. E.g.

curl -o "name.jpeg" "http://legacysurvey.org/viewer/jpeg-cutout?ra=334.62567&dec=0.20588&size=512&layer=dr8&pixscale=0.262&bands=grz"

where the (pixel) size should be chosen to correspond the size of the discovery image postage stamp area.

### PR DESCRIPTION
**Pull request by acrellin**
Closes #217

### CODE REVIEW

**Comment by acrellin** [comment]
@stefanv this should be good to go!

File: skyportal/models.py [hunk 1]
```diff
@@ -147,36 +147,24 @@ class Source(Base):
...
-                              type='ps1')
-        DBSession().add_all([sdss_thumb, ps1_thumb])
+        dr8_thumb = Thumbnail(photometry=self.photometry[0],
+                              public_url=self.desi_dr8_url(),
+                              type='dr8')
+        DBSession().add_all([sdss_thumb, dr8_thumb])
         DBSession().commit()
 
-    def get_sdss_url(self):
+    def sdss_url(self):
```
Comment (stefanv) [inline skyportal/models.py]: These still need an `@property` above.  And then removal of the `()` in all calls elsewhere.

Comment (acrellin) [reply skyportal/models.py]: Sure, done.

**Comment by stefanv** [comment]
In it goes!

**Pull request merged**